contents/guides/users/aspects.md

```diff
@@ -0,0 +1,41 @@
+---
+title: What are aspects and how do I use them?
+category: User guides
+---
+
+## What is an aspect?
+
+Think of your life and the people you know. Each person is a part of one or more aspects of your life. They might be a member of your family, a close friend, a work colleague or someone you play sport or music with or with whom you share a particular interest. They might be someone you just like sharing jokes with online. Or they might be part of more than one of these aspects of your life.
+
+diaspora\* works in exactly the same way: Aspects are the way you group your contacts on diaspora\*.
+
+You place contacts into one or more “aspects” of your life on diaspora\* based on which aspect(s) of your life they are part of. In this way, you control which of your contacts sees each post you make to diaspora\*, by posting to a particular aspect or aspects. This means that you can easily share the right things with the right people.
+
+## How can I manage my aspects?
```

Comment:
I've done major work from here on, so I hope you don't mind if I just paste it in one comment. I've reordered some paragraphs, starting with the 'When you sign up' para moved before this heading:

When you sign up you are given four default aspects: Family, Work, Friends and Acquaintances. Of course you won’t be able to sort all of your life into just four standard aspects, so you can add some aspects. You can also delete the default aspects if you want to.

## How can I manage my aspects?

Manage your aspects from your Contacts page. To access this, use the drop-down menu at the top right and select *Contacts*. You can also manage your aspects by clicking “My aspects” in the left side-bar from a stream view.

### How do I add or delete aspects?

#### Add an aspect

Click **+ Add an aspect** under the list of aspects in the menu. You will be presented with a pop-up window. Choose a name and click “Create”.

Your contacts will never see the name of the aspect you put them in so you don’t need to worry about what you call your aspects.

#### Rename an aspect

Select the relevant aspect and click the edit icon next to the aspect name at the top of this page, change the name and press “Update”.

#### Delete an aspect

Select the relevant aspect and click the trash icon in the top right of the page.

### How do I add contacts or remove them from my aspects?

To add someone as a contact or to change which of your aspects they are in, simply hover over their user name wherever you see it. You can then select the aspects you want them in using the “hover-card” that pops up. If they were not previously in any of your aspects, they'll receive a notification that you have started to share with them.

Alternatively, go to your Contacts page. Use the menu to the right of each contact to add them to or remove them from aspects. Or you can use the aspect selector button on their profile page.

You can stop sharing with someone by removing them from all your aspects. They don't receive a notification when this happens.

You can read more about at “[How do I manage my contacts and sharing?][contact_management_and_sharing]”

## How do I publish posts to aspects?

Use the “aspect selector” button in the publisher before publishing your post to choose which aspects you will share your content with. Posts sent to aspects are always private and cannot be reshared.

Once you have published your post, you can't change which aspects it is shared with. People who were not in the relevant aspects when you published your post won't receive it, even if you add them to that aspect later.

You can set the default visibility for your posts – one or more aspects, all aspects or public – in your account settings.

Reply:
I like your changes except for the very short paragraphs under "How do I add or delete aspects?" I don't think the extra titles are necessary as there's only very little text in each section. Big headings make it harder to read (and it doesn't look nice). 
I also think your explanations are a little too short here. it's easier to follow the explanations if there's a bit more text with more detailed descriptions for every single step. 
What do you think of my updated version?